### PR TITLE
Feature/patients list sort fixes

### PR DESCRIPTION
**Pull request by achen2401**
found these issues while testing:
fix sorting issue for ID (numeric) and study ID (alphanumeric) fields
for id column - remove sorter attribute as anchor link is no longer present in the field
for study id column - add alpha numeric sorter for sorting

### CODE REVIEW

File: portal/static/js/main.js [hunk 1]
```diff
@@ -4222,6 +4222,42 @@ var tnthTables = {
...
 
         return  b_d - a_d;
+    },
+    /***
+     * sorting alpha numeric string
+     */
+     "alphanumericSorter": function (a, b) {
+        /*
+         * ideas came from
+         * http://www.davekoelle.com/files/alphanum.js
```
Comment (ivan-c) [inline portal/static/js/main.js]: Would it make sense to keep this as a separate file? I found it copied as [a gist here](https://gist.github.com/myadzel/6405e60256df579eda8c). looks like those work with rawgit too :)

https://cdn.rawgit.com/myadzel/6405e60256df579eda8c/raw/e24a756e168cb82d0798685fd3069a75f191783f/alphanum.js

File: portal/templates/patients_by_org.html [hunk 2]
```diff
@@ -48,7 +48,7 @@ <h4 class="tnth-headline">Patient List</h4>
               <tr>
                   <th data-field="id" data-visible="false" data-card-visible="false" data-sortable="false" data-class="tnth-hide">
                   <!-- need to hide the ID column specifically for EPROMs -->
-                  <th data-field="userid" data-sortable="true" data-class="id-field" data-sorter="tnthTables.stripLinksSorter" data-width="1%" {%if config.HIDE_TRUENTH_ID_FIELD%}data-visible="false"{%endif%}>TrueNTH ID</th>
+                  <th data-field="userid" data-sortable="true" data-class="id-field" data-width="1%" {%if config.HIDE_TRUENTH_ID_FIELD%}data-visible="false"{%endif%}>TrueNTH ID</th>
```
Comment (ivan-c) [inline portal/templates/patients_by_org.html]: Could you tag `TrueNTH ID`?

**Comment by achen2401** [comment]
@ivan-c Ivan, I think I addressed all of your requests.  But let me know if you see any issue with my changes.

**Comment by ivan-c** [comment]
Will merge once tests pass...

**Pull request merged**